src/packages/excalidraw/README.md

```diff
@@ -643,6 +643,7 @@ The `<Sidebar>` component takes these props (all are optional except `children`)
 | onDock | <pre>(isDocked: boolean) => void</pre> | Invoked when the user toggles the dock button. |
 | docked | boolean | Indicates whether the sidebar is docked. By default, the sidebar is undocked. If passed, the docking becomes controlled, and you are responsible for updating the `docked` state by listening on `onDock` callback. See [here](#dockedSidebarBreakpoint) for more info docking. |
 | dockable | boolean | Indicates whether the sidebar can be docked by user (=the dock button is shown). If `false`, you can still dock programmatically by passing `docked=true` |
+| ref | <pre>React.Ref&lt;HTMLDivElement&gt;</pre> | forward ref |
```

Comment:
Since this going to be removed in future PR, lets not add it here

Reply:
it's likely not gonna get removed. Refs in exported components are a good idea in general. It's just we may not use it internally.